app/package.json

```diff
@@ -5,7 +5,6 @@
 	"homepage": "https://purecloudlabs.github.io/genesys-react-components",
 	"dependencies": {
 		"genesys-dev-icons": "^0.1.30",
-		"genesys-react-components": "^0.3.2",
```

Comment:
Make sure you run `npm run unlink` in this dir to revert the local link and add the normal dependency back to the config. This leaves the project in a usable state for other users that may not have the local link set up.